mautic/mautic
# Change '0 mn' option to 'Never retry' since this is the current behavior

closes #8882 
When use sets '0 mn' in Settings -> Configuration -> Campaign Settings,
failed actions are never retried. Change the label to match the current
behavior.

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | No
| Automated tests included? |No
| Related user documentation PR URL | No
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | #8882 
| BC breaks? | It shouldn't break anything
| Deprecations? | N/A

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

When use sets _Settings -> Configuration -> Campaign Settings -> Wait time before retrying a failed action_ to `0 mn`, the action are never retried. Is possible to find more information [here](https://github.com/mautic/mautic/issues/8882#issuecomment-639772887).

This PR simply changes the label from `0 mn` to `Never retry` which is the current behavior. This should be enough to avoid user confusion as I get confused. 

[//]: # ( As applicable: )

#### Steps to reproduce the bug:

From #8882 

1. Install the plugin from https://github.com/dhilst/HelloWorldBundle
2. Go to Setup -> Configuration -> Campaign Settings and set _Wait time before retrying a failed action_ to 0mn
3. Create some contacts
4. Create a segment with that contacts
5. Create a campaign with that segment and next the _HelloWorld - Blast off_ action
6. Tail the log in another terminal `tail -F app/logs/mautic_dev-$(date +%Y-%m0-%d).php`
7. At first terminal run `app/console mautic:campaign:update --env=dev && app/console mautic:campaign:trigger --env=dev` (I'm using dev environment to test the plugin)
8. At the terminal running tail is check that the action was executed by presence of `[2020-06-05 13:54:40] mautic.INFO: ==> BLASTOFF 2020-06-05T13:54:40+00:00` message (the date will change). Press some returns to split the output for the next run
9. At first terminal run `app/console mautic:campaign:update --env=dev && app/console mautic:campaign:trigger --env=dev` again
10. At the terminal running tail no `==> BLASTOFF 2020-06-05T13:54:40+00:00` message, meaning that the action was not tried again.

#### Steps to test this PR:

1. Just apply the PR and navigate to _Settings -> Configuration -> Campaign Settings_. The label should be visible 

#### List deprecations along with the new alternative:

N/A

#### List backwards compatibility breaks:

N/A
